PresetParser/Anno1800/NewFactionAndGroup1800.cs

```diff
@@ -13,7 +13,7 @@ namespace PresetParser.Anno1800
     /// ChangeBuildingTo<1>_<2>_1800 
     /// <1> can be : OW (All Worlds) - OW1 (New World - (1) Farmers) - OW2 (New World - (2) Workers) - OW3 (New World - (3) Artisans)
     ///              OW4 (New World - (4) Engineers  - OW5 (New World - (5) Investors) - OW6 (New World (13) Scholars)  
-    ///              NW1 (Old World - (7) Jornaleros  - NW2 (Old World - (8) Obreros)
+    ///              NW1 (Old World - (7) Jornaleros  - NW2 (Old World - (8) Obreros  - NW3 (Old World - (21) Artista)
```

Comment:
Somehow feels kind of off to have `(7) Jornaleros` and `(8) Obreros` in the menu right next to each other, and then way later `(21) Artista`, but `(9) ...` was already taken :man_shrugging: